.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Type of change
+
+_Select the type of your PR_
+
+- [ ] Bugfix
+- [ ] Enhancement / new feature
```

Comment:
Actually, I just realised that this checkbox list comes with an unfortnate side-effect. On the [PRs page](https://github.com/strimzi/strimzi/pulls) it will show the progress of a PR in terms of the number of checked boxes out of the total number of checkboxes. Because this list is really an either/or, that progress doesn't reach 8/8 when all the work has been done; instead it stays at 7/8. Perhaps it's worth making this a non-checkbox so that that progress on the PRs page isn't broken.